### ISSUE TITLE
From old 'Documentation Admin' page

### ISSUE THREAD

**Issue by hdgarrood** [purescript/documentation#6]
Things that people wanted documenting:

- Newtype
-- Difference between keyword and type class.
-- Removed `runAdditive` in v0.10, moved to `wrap` from `Newtype` class. Why have `run*` in first place?

- Functional Dependencies
-- What type-checking errors does it resolve?
-- When should I use functional dependency in my type class?

- Typed-holes and type wildcards
-- Names are similar, needs clarification.

- Compile Iteration Techniques
-- Hole-driven development with typed-holes.
-- Faculties of editor plugins. Using pscid as editor-independent alternative.
-- Iterating module while testing in psci.

- Generics
-- How to use generics to `encodeJson` from argonaut?
-- Old generics (purescript-generics) vs new generics (purescript-generics-rep, enabled by fun-deps)

**Comment by soupi** [comment]
Note that some of those are answered in The PureScript 0.10 release video on youtube ([link](https://www.youtube.com/watch?v=CMqw09wQUvQ)).

**Comment by hdgarrood** [comment]
Thanks for the reminder! I think we should aim to have that information somewhere in this repository too.

**Comment by chexxor** [comment]
Still need to translate to text, though.

**Comment by soupi** [comment]
Definitely! But it can help anyone who is interested in writing docs about it :)
